packages/algob/src/internal/deployer.ts

```diff
@@ -194,6 +195,17 @@ class DeployerBasicMode {
     return await getLsigFromCache(name);
   }
 
+  /**
+   * Returns program compiled info: bytecode (compiled program) and it's hash.
+   * @param name ASC name
+   * @param scTmplParams: Smart contract template parameters (used only when compiling PyTEAL to TEAL)
+   * @returns loaded logic signature from artifacts/cache/<file_name>.teal.yaml
+   */
+  async getCompiledASCInfo (name: string,
```

Comment:
Maybe we can remove `Info` from the name?

Reply:
ok